ui/src/timespan/TimeSpan.tsx

```diff
@@ -16,14 +16,13 @@ import Menu from '@material-ui/core/Menu';
 import MenuItem from '@material-ui/core/MenuItem';
 import {RemoveTimeSpan, RemoveTimeSpanVariables} from '../gql/__generated__/RemoveTimeSpan';
 import {useStateAndDelegateWithDelayOnChange} from '../utils/hooks';
```

Comment:
Mhmm, I think travis ci finally killed the open source builds or this project doesn't have credits left.

Reply:
Tragic. Probably time to move to GitHub Actions though anyway.